src/api/json/catalog.json

```diff
@@ -2,6 +2,15 @@
   "$schema": "https://json.schemastore.org/schema-catalog.json",
   "version": 1,
   "schemas": [
+    {
+      "name": "cff",
+      "description": "Citation File Format",
+      "fileMatch": [".cff"],
+      "url": "https://raw.githubusercontent.com/citation-file-format/citation-file-format/main/schema.json",
+      "versions": {
+        "1.2.0": "https://raw.githubusercontent.com/citation-file-format/citation-file-format/1.2.0/schema.json"
+      }
+    },
```

Comment:
```suggestion
```

Adds a duplicate to the entry in L1575ff. Please update entry there instead.

Also note that valid CFF files should be named `CITATION.cff`. Suggested updated entry in ll. 1575 ff.:

```json
    {
      "name": "Citation File Format",
      "description": "A YAML file with citation metadata for software or datasets",
      "fileMatch": ["CITATION.cff"],
      "url": "https://raw.githubusercontent.com/citation-file-format/citation-file-format/main/schema.json",
      "versions": {
        "1.2.0": "https://raw.githubusercontent.com/citation-file-format/citation-file-format/1.2.0/schema.json"
      }
    },
```

Reply:
Entry in current version:

https://github.com/SchemaStore/schemastore/blob/310268503357b43b0780ae6bca16627cf6182980/src/api/json/catalog.json#L1566-L1571